lib/Indexer/Extension/Command/IndexBuildCommand.php

```diff
@@ -73,38 +73,41 @@ private function buildIndex(OutputInterface $output, ?string $subPath = null): v
         $start = microtime(true);
 
         $output->write('<info>Building job</info>...');
-        $job = $this->indexer->getJob($subPath);
-        $output->writeln('done');
-        $output->writeln('<info>Building index:</info>');
-        $output->write(PHP_EOL);
-
-        if ($job->size() === 0) {
-            $output->writeln('No files found');
-            return;
-        }
+        $jobs = $this->indexer->getJobs($subPath);
```

Comment:
Bit of a smell as sub path can only apply to a single index (i.e. the "project" index)